packages/components/src/DateTimePickers/DatePicker/CalendarHeader/CalendarHeader.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import theme from './CalendarHeader.scss';
+
+function CalendarHeader(props) {
+	const dayNames = ['L', 'M', 'M', 'J', 'V', 'S', 'D'];
+	return (
+		<div className={theme.container}>
+			{dayNames.map((dayName, i) =>
+				<abbr className={theme['day-of-week']} key={i}>{dayName}</abbr>
```

Comment:
Just to have in mind, we will need to have the full name (in aria-label/title).

Reply:
&#10003;